### PR TITLE
Shadow cube for point light

### PR DESCRIPTION
**Pull request by ryanastout**
Code to create a shadow cubemap for a point light. I added a test ShadowMappingPointLightTest.java and verified that ShadowMappingTest.java still behaves correctly.

### CODE REVIEW

File: gdx/src/com/badlogic/gdx/graphics/g3d/environment/PointShadowLight.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+ *		modelBatch.render(instance, environment);
+ *		modelBatch.end();
+ * 
+ * @deprecated Experimental, likely to change, do not use!
+ * @author ryanastout */
+public class PointShadowLight extends PointLight implements Disposable {
+	protected FrameBufferCubemap frameBufferCube;
+	protected Camera camera;
+	protected TextureDescriptor textureDesc;
+	private ModelBatch modelBatch;
```
Comment (Tom-Ski) [inline gdx/src/com/badlogic/gdx/graphics/g3d/environment/PointShadowLight.java]: Should be flexible to use without ModelBatch.

**Comment by Tom-Ski** [comment]
test this please

**Comment by ryanastout** [comment]
@Tom-Ski Any suggestions on how to make this better?

**Comment by Darkyenus** [comment]
> Is there any reason for me not to delete this PR?

Even when it is not merged, someone may be able to use it by merging it locally.

Maintainers are volunteers and sadly don't have time to review larger PRs (and I should know, one of mine is the second oldest unresolved PR...).

**Comment by ryanastout** [comment]
At this point there are conflicts so I wasn't sure it'd still be useful to anyone since it can't just be cherry-picked as is.  But if it's useful to someone even just as a rough guide to shadows, I don't mind leaving it up. I won't bother with fixing the conflicts, though.

**Pull request closed without merging**